src/util/simplify_expr_int.cpp

```diff
@@ -53,6 +54,114 @@ bool simplify_exprt::simplify_bswap(bswap_exprt &expr)
   return true;
 }
 
+static bool sum_expr(exprt &dest, const exprt &expr)
```

Comment:
Add doxy comment, e.g. does this intend to add typecasts if needed, can it do pointer arithmetic, ...